README.md

```diff
@@ -77,3 +77,10 @@ And run the tests:
 ```bash
 poetry run pytest
 ```
+
+## Vagrant
+You can run this system in vagrant, for that install vagrant, virtual box and run:
+And run the tests:
+```bash
+vagrant  up --provision
```

Comment:
In normal running, we don't want the user to run with `--provision` every time - vagrant will automatically provision the first time that the VM is run, and afterwards they shouldn't need to re-provision (and avoiding doing so will speed up spinning up the VM)